### ISSUE TITLE
Tracking. canvassers 

### ISSUE THREAD

**Issue by KenStanley** [DaleStan/VoterMate#39]
mnivan also keeps track of which canvassers performed the canvass. 

Could we have an initial screen that allowed me to enter the canvassers name, or perhaps their SOS_VOTERID? Maybe the same screen where I can assign a turf to them.

**Comment by DaleStan** [comment]
Where do you want the output to record the canvasser? File name, the first column of the csv, and the last column of the csv all seem like reasonable options.

**Comment by KenStanley** [comment]
i think the first column would be ideal.. wood or new possible to add a
header column? I am neighbors about changing the column meanings without
headings.

Ken

**Comment by KenStanley** [comment]
Wow I obviously wrote this from my phone and did not re-read the comments. The other option being that i was drunk as a skunk, which I was not. Let me try again:

I think the first column would be ideal.
Would it be possible to add a header column? I am nervous about changing the column meanings without
headings.

**Comment by KenStanley** [comment]
Actually, if you add column headings, it doesn't matter which column it is.
